tests/Unit/ProxyClient/MultiplexerClientTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Tests\Unit\ProxyClient;
+
+use FOS\HttpCache\ProxyClient\Invalidation\BanInterface;
+use FOS\HttpCache\ProxyClient\Invalidation\RefreshInterface;
+use FOS\HttpCache\ProxyClient\MultiplexerClient;
+use FOS\HttpCache\ProxyClient\ProxyClientInterface;
+
+class MultiplexerClientTest extends \PHPUnit_Framework_TestCase
+{
+    public function testBan()
+    {
+        $headers = ['Header1' => 'Header1-Value'];
+
+        $mockClient1 = $this->createMock(BanInterface::class);
```

Comment:
we still support php 5.5 and that means phpunit 4.* which means no createMock :-(

can you please use the old getMock instead?